sample-app-2/build.gradle

```diff
@@ -1,19 +1,17 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    compileSdkVersion parent.ext.androidCompileSdkVersion
 
 
     defaultConfig {
-        applicationId "com.madmaclon.sample_app_2"
-        minSdkVersion 16
-        targetSdkVersion 26
+        applicationId "com.madmaclon.sampleapp2"
```

Comment:
Would it be a good opportunity to change the application id/package name for something more yotian?